index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
```

Comment:
is your stuff using spaces not tabs, and using 2 spaces instead of 4? I'm using only spaces and 2 spaces for each nest, which means if you're using tabs or 4 spaces there could be compatibility issues later.